sycl/include/sycl/detail/pi.def

```diff
@@ -141,6 +141,20 @@ _PI_API(piextKernelSetArgSampler)
 
 _PI_API(piextPluginGetOpaqueData)
 
+/// command-buffer Extension
+_PI_API(piextCommandBufferCreate)
+_PI_API(piextCommandBufferRetain)
+_PI_API(piextCommandBufferRelease)
+_PI_API(piextCommandBufferFinalize)
+_PI_API(piextCommandBufferNDRangeKernel)
+_PI_API(piextCommandBufferMemcpyUSM)
+_PI_API(piextCommandBufferMemBufferCopy)
+_PI_API(piextCommandBufferMemBufferCopyRect)
+_PI_API(piextCommandBufferMemBufferWrite)
+_PI_API(piextCommandBufferMemBufferWriteRect)
+_PI_API(piextCommandBufferMemBufferRead)
+_PI_API(piextCommandBufferMemBufferReadRect)
+_PI_API(piextEnqueueCommandBuffer)
```

Comment:
Could you please add these to the end of the list of API definitions? See the comment on line 17.

Reply:
Good point, missed that advice on line 17. Updated.